framework/wazuh/configuration.py

```diff
@@ -452,24 +453,24 @@ def get_ossec_conf(section=None, field=None, conf_file=common.ossec_conf):
         # Parse XML to JSON
         data = _ossecconf2json(xml_data)
     except Exception as e:
-        raise WazuhException(1101, str(e))
+        raise WazuhError(1101, extra_message=str(e))
```

Comment:
We are returned e directly, this might return private info about the full_path, which is something we need to avoid.

Reply:
Done!